docs/reference/cli/index.md

```diff
@@ -3422,6 +3422,63 @@ You can overwrite the file while Teku is running to update the graffiti.
 
 This option takes precedence over [`--validators-graffiti`](#validators-graffiti).
 
+### `validators-graffiti-client-append-format`
+
+<Tabs>
+  <TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--validators-graffiti-client-append-format=<STRING>
+```
+
+  </TabItem>
+  <TabItem value="Example" label="Example" >
+
+```bash
+--validators-graffiti-client-append-format=CLIENT_CODES
+```
+
+  </TabItem>
+  <TabItem value="Environment variable" label="Environment variable" >
+
+```bash
+TEKU_VALIDATORS_GRAFFITI_CLIENT_APPEND_FORMAT=CLIENT_CODES
+```
+
+  </TabItem>
+  <TabItem value="Configuration file" label="Configuration file" >
+
+```bash
+validators-graffiti-client-append-format: CLIENT_CODES
+```
+
+  </TabItem>
+</Tabs>
+
+Appends CL and EL clients information with a space to user's graffiti. On
+a separate BN/VC setup should be set on a Beacon Node. This feature helps
+developers and community to analyze client diversity and block anomalies.
+
+Default value is `AUTO`.
+
+Following options are available:
+- `AUTO`: If user's graffiti is empty sets graffiti to reveal
```

Comment:
Maybe this can be rewritten in a slightly different way to  for improve clarity, for example:

`AUTO`: If a user's graffiti is empty, it automatically updates to reveal information about the consensus and execution clients. This includes their codes and build commits, for example `TK508459f2BUbb9ba13c`, with the structure as follows:
   - `TK` represents the Teku consensus client.
   -  `508459f2` is the Teku build commit.
   -  `BU` indicates the Besu execution layer client.
   -  `bb9ba13c` displays the Besu build commit in a 4-byte hexadecimal format.

Reply:
I like it, updated